src/components/Editor.jsx

```diff
@@ -136,12 +136,60 @@ class Editor extends React.Component {
     session.on('change', () => {
       this.props.onInput(this._editor.getValue());
     });
+    session.selection.on('changeCursor', () => {
+      let selector = '';
+      if (this.props.language === 'css') {
+        selector = this._locateSelectorCSS(session);
+      }
+      this._sendSelectorToHighlighter(selector);
+    });
     session.setAnnotations(this.props.errors);
     this._editor.setSession(session);
     this._editor.moveCursorTo(0, 0);
     this._resizeEditor();
   }
 
+  _locateSelectorCSS(session) {
```

Comment:
I would propose moving this logic into a separate module, probably just a function that takes `lines` and `cursor` parameters.

Reply:
This sounds good! Where would the file for this module live?

Reply:
Probably just in `util`, the junk drawer for useful but hard-to-categorize modules : )